.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -0,0 +1,50 @@
+param ($GitHub)
+
+switch ($GitHub.event_name)
+{
+    "discussion"
+    {
+        $summary = "Respond to `"$($GitHub.event.discussion.title)`""
+        $description = $Env:DISCUSSION_JIRA_ISSUE_DESCRIPTION
+        $link = $GitHub.event.discussion.html_url
+    }
+    "issues"
+    {
+        $summary = "$($GitHub.event.issue.title) in $($GitHub.repository)"
```

Comment:
Shouldn't this be quoted too? Alternatively the `in $($GitHub.repository)` could be parenthesized.

Reply:
Do you mean quotes around `$($GitHub.repository)`? Since it's a clearly self-contained text (just as the component name), unlike a PR or discussion title, it didn't feel right.

Reply:
I mean the other summaries are like ``"Respond to `"$($GitHub.event.discussion.title)`""``, so I wondered if this should be ``$summary = "`"$($GitHub.event.issue.title)`" in $($GitHub.repository)"`` as well.

The alternative I mentioned would be adding a parenthesis around the repo.
So in the new code `$titleSuffix = " (in $context)"`.

Reply:
I see. It's not `$summary = "`"$($GitHub.event.issue.title)`" in $($GitHub.repository)"` because, unlike the other two events, we're opening a Jira issue for a GitHub **issue**, so it's adequate to use the same title. Note that while in the case of issues the GH title is part of the title proper, in the other two cases the PR/discussion titles are references _from_ the title:

![image](https://user-images.githubusercontent.com/1976647/190684279-cd7d1791-b8f4-4f5c-a5e0-c8d766654d33.png)

The parenthesis, I'm not sure. It doesn't seem to be necessary for me.